src/main/java/org/llorllale/cactoos/matchers/ScalarHasValue.java

```diff
@@ -82,6 +82,6 @@ protected boolean matchesSafely(final Scalar<T> item) {
     protected void describeMismatchSafely(final Scalar<T> item,
```

Comment:
What so-called `refused bequest` here, `describeMismatchSafely` needs to call super.